Rakefile

```diff
@@ -4,3 +4,11 @@
 require_relative 'config/application'
 
 Rails.application.load_tasks
+
+namespace :cf do
```

Comment:
We should put this with the rest of the rake tasks at `lib/tasks`

Reply:
moved!